hail/python/hail/ir/register_functions.py

```diff
@@ -1,18 +1,18 @@
 import hail as hl
 from hail.expr.nat import NatVariable
-from .ir import register_function, register_session_function, register_seeded_function
+from .ir import register_function, register_seeded_function
 
 
 def register_reference_genome_functions(rg):
```

Comment:
This shouldn't depend on `rg`.  These functions shoudl be registered once in `register_functions` same as Scala.  The `tlocus(rg)` should be replaced by a type variable with a `locus` condition.  The call to this needs to be removed from the reference genome constructor.